src/main/java/com/expense/Sulaah/controller/TransactionController.java

```diff
@@ -29,9 +29,9 @@ private ResponseEntity<?> getTransactionsInGroup(@PathVariable int id) {
 	}
 
 	@PostMapping("/")
-	private Transaction addTransaction(@RequestBody TransactionDto transaction) {
-		return transactionService.addTransaction(transaction);
-
+	private Transaction addTransaction(@RequestBody TransactionDto transactionDto) {
+//		groupService.addMembers(transaction.getGroupId(),transaction.getUserIdWithShare().keySet().stream().toList());
```

Comment:
remove this comment.
adding members from a transaction is not logically correct

Reply:
Yeah, changed in a new commit